aws_xray_sdk/ext/django/templates.py

```diff
@@ -22,6 +23,10 @@ def xray_render(self, context):
         template_name = self.name or getattr(context, 'template_name', None)
         if template_name:
             name = str(template_name)
+            # SafeString are not properly serialized by jsonpickle,
+            # turn them back to str by adding a non-safe str.
+            if isinstance(name, SafeString):
```

Comment:
Can you add a test case for this code path to maintain coverage?

Reply:
I added the test and updated the django version (the test wouldn't cover the case without updating django).